Sources/Sentry/SentrySessionReplayIntegration.m

```diff
@@ -351,6 +351,35 @@ - (void)resume
     [self.sessionReplay resume];
 }
 
+- (void)start
+{
+    if (self.sessionReplay != nil) {
+        if (self.sessionReplay.isFullSession == NO) {
+            [self.sessionReplay captureReplay];
+        }
+        return;
+    }
+
+    _startedAsFullSession = YES;
+    if (SentryDependencyContainer.sharedInstance.application.windows.count > 0) {
+        // If a window its already available start replay right away
+        [self startWithOptions:_replayOptions fullSession:YES];
+    } else {
+        // Wait for a scene to be available to started the replay
+        if (@available(iOS 13.0, tvOS 13.0, *)) {
+            [_notificationCenter addObserver:self
+                                    selector:@selector(newSceneActivate)
+                                        name:UISceneDidActivateNotification];
+        }
+    }
```

Comment:
`m`: That's duplicate code of `startSession`. Maybe we can reuse some code.

https://github.com/getsentry/sentry-cocoa/blob/913fab76543f3893b2e1f620546aa2e0636c3102/Sources/Sentry/SentrySessionReplayIntegration.m#L215-L225